src/test/java/myfirstproject/service/impl/ReadFileImplTest.java

```diff
@@ -0,0 +1,50 @@
+package myfirstproject.service.impl;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import myfirstproject.service.ReadFile;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReadFileImplTest {
+    private static final String PATH_TO_NEW_FILE = "src/test/resources/sourceFile.csv";
+    private static final String FILE_NOT_EXIST = "wrong/file.csv";
+    private static ReadFile fileReaderTest;
+    private static List<String[]> temporalList;
+
+    @BeforeClass
+    public static void before() {
+        temporalList = new ArrayList<>();
+        fileReaderTest = new ReadFileImpl();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void readWrongFile_notOk() {
+        fileReaderTest.readFile(temporalList, FILE_NOT_EXIST);
+    }
+
+    @Test
+    public void readFile_Ok() {
```

Comment:
```suggestion
    public void readFile_fileExists_ok() {
```
check naming everywhere